flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/application/SqlDriver.java

```diff
@@ -0,0 +1,223 @@
+        SqlDriver.testMode = false;
+    }
+
+    private static ClassLoader getClassLoader() throws Exception {
+        MutableURLClassLoader sqlGatewayClassloader =
+                (MutableURLClassLoader) Thread.currentThread().getContextClassLoader();
+        try {
+            sqlGatewayClassloader.loadClass(RUNNER_CLASS_NAME);
+            LOG.info("Load {} from the classpath.", RUNNER_CLASS_NAME);
+        } catch (ClassNotFoundException e) {
+            LOG.info("{} is not in the classpath. Finding...", RUNNER_CLASS_NAME);
+            sqlGatewayClassloader.addURL(findExecutor().toUri().toURL());
+        }
+        return sqlGatewayClassloader;
+    }
+
+    private static Path findExecutor() {
+        String flinkOptPath = System.getenv(ConfigConstants.ENV_FLINK_OPT_DIR);
+        final List<Path> sqlJarPaths = new ArrayList<>();
+        try {
+            Files.walkFileTree(
+                    FileSystems.getDefault().getPath(flinkOptPath),
+                    new SimpleFileVisitor<>() {
+                        @Override
+                        public FileVisitResult visitFile(Path file, BasicFileAttributes attrs)
+                                throws IOException {
+                            FileVisitResult result = super.visitFile(file, attrs);
+                            if (file.getFileName().toString().startsWith("flink-sql-gateway")) {
+                                sqlJarPaths.add(file);
+                            }
+                            return result;
+                        }
+                    });
+        } catch (IOException e) {
+            throw new RuntimeException(
+                    "Exception encountered during finding the flink-sql-gateway jar. This should not happen.",
+                    e);
+        }
+
+        if (sqlJarPaths.size() != 1) {
+            throw new RuntimeException("Found " + sqlJarPaths.size() + " flink-sql-gateway jar.");
+        }
+
+        return sqlJarPaths.get(0);
+    }
+
+    static Options getSqlDriverOptions() {
+        Options options = new Options();
+        options.addOption(OPTION_SQL_FILE);
+        options.addOption(OPTION_SQL_STATEMENTS);
+        return options;
+    }
+
+    // --------------------------------------------------------------------------------------------
+    //  Line Parsing
+    // --------------------------------------------------------------------------------------------
+
+    public static String parseOptions(String[] args) {
+        try {
+            DefaultParser parser = new DefaultParser();
+            CommandLine line = parser.parse(getSqlDriverOptions(), args, true);
+            String content = getContent(line.getOptionValue(OPTION_SQL_FILE.getLongOpt()));
+            if (content == null) {
+                return Preconditions.checkNotNull(
+                        line.getOptionValue(OPTION_SQL_STATEMENTS.getLongOpt()),
+                        "Please set script or script path either.");
```

Comment:
Modify the prompt to make it clearer, suggesting the use of --script or --scriptPath to set the parameters?